simple_test.sh

```diff
@@ -1,48 +1,81 @@
 #!/bin/sh
 
+# This test script automates our distributed work system. When running this script, please run it as follows:
+# ./simple_test [hostname] [# of workers] [test filename]
+#
+# [hostname] refers to the complete address with "http://" or "https://" preceding it.
+# [test filename] refers to the test file located in the test directory.
```

Comment:
Adding @tuckerzp to this conversation

Do we really want to force users to choose a script here?

Reply:
No. I requested changed below @showalter